src/solver/conv_ocl_dir2Dfwd1x1.cpp

```diff
@@ -36,6 +36,8 @@
 /// Known since ROCm 2.9.
 #define WORKAROUND_ISSUE_2298 1
 
+#define WORKAROUND_SWDEV_271887 1
```

Comment:
@zjing14 On April 7 2021 https://ontrack-internal.amd.com/browse/SWDEV-271887 was set to implemented state. It needs to find out ROCm release the fix is included in, and disable the W/A starting from that release. Then we can check that the issue is gone and close the Jira.